layers/meta-opentrons/recipes-robot/opentrons-user-environment/files/opentrons_simulate

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+export RUNNING_ON_VERDIN=1
+export OT_API_FF_enableOT3HardwareController="true"
```

Comment:
should this be false for simulate?

Reply:
No, this is always true - this put it into "flex mode", it's not about controller vs simulator but about ot2 controller vs flex controller